transformations/abbreviation_transformation/abbrev.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+"""
+ref: https://medium.com/coinmonks/remaking-of-shortened-sms-tweet-post-slangs-and-word-contraction-into-sentences-nlp-7bd1bbc6fcff
+"""
+
+from bs4 import BeautifulSoup

```

Comment:
If the code is not being used directly, you might want to comment it

Reply:
Just to clarify, this means to comment the code out in this file, right? Not comment it in a different file.

Reply:
What you've done is perfectly fine. :)